tapir/shifts/views/management.py

```diff
@@ -110,9 +111,11 @@ class CancelDayShiftsView(LoginRequiredMixin, PermissionRequiredMixin, FormView)
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         day = datetime.datetime.strptime(self.kwargs["day"], "%d-%m-%y").date()
-        kwargs["shifts"] = Shift.objects.filter(start_time__date=day).order_by(
-            "start_time"
-        )
+        filter_day = Q(start_time__date=day)
+        filter_not_deleted = Q(deleted=False)
+        kwargs["shifts"] = Shift.objects.filter(
+            filter_day & filter_not_deleted
+        ).order_by("start_time")
```

Comment:
```suggestion
        kwargs["shifts"] = Shift.objects.filter(
            start_time__date=day, deleted=False
        ).order_by("start_time")
```

`Q`-Objects are only needed when doing filters with `OR`, with `AND` you can just put both fields in `filter()`